readme.txt

```diff
@@ -356,6 +356,7 @@ At no point during the 3.0 lifecycle will the major version change. But you can
 * Fix - Fixed a problem where iCal data was generated with incorrect dates in the case of some all days events (thanks to Matt for highlighting this)
 * Fix - Resolved a problem causing the previous month view to appear when it should not
 * Fix - Fixed issue in mobile month view where date was missing from heading
+* Fix - Resolved issue that caused /events/ to 404 if it was visited on a new install before hitting "Save" on the Events > Settings > Display page
```

Comment:
was this an existing bug that we fixed, or something caused by the work in 4.0 (in other words, is a changelog entry required here?)